### PR TITLE
Create ADO-ID-to-TP-URL

### PR DESCRIPTION
**Pull request by nadiab**

### CODE REVIEW

File: ADO JS Mappings/ADO-ID-to-TP-URL [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Mapping of TP URL CF to ADO ID
```
Comment (roman-kozachenko) [inline ADO JS Mappings/ADO-ID-to-TP-URL]: Could you please add `.md` extension to the file?

File: ADO JS Mappings/ADO-ID-to-TP-URL [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
+From TP to ADO
```
Comment (roman-kozachenko) [inline ADO JS Mappings/ADO-ID-to-TP-URL]: You can add some structure for the document, for example transform that text to the header with `###`.
That will increase readability and allow us to send links for particular document part, for example: https://github.com/TargetProcess/JS-mapping-templates/blob/6c8473c11529cc337acc76624c122942cc268929/ADO%20JS%20Mappings/art-to-area-path.md#ado--tp

File: ADO JS Mappings/ADO-ID-to-TP-URL [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  }
+}
+```
+
+
+COMPARATOR
+
+```
+const {sourceFieldValue, targetFieldValue} = args;
+// const url = sourceFieldValue.toolValue && sourceFieldValue.toolValue.URL;
```
Comment (roman-kozachenko) [inline ADO JS Mappings/ADO-ID-to-TP-URL]: Does these comments make any sense here?
Maybe it can be removed?

**Review by roman-kozachenko — CHANGES_REQUESTED** [review]

**Comment by nadiab** [comment]
updated as suggested in comments

**Pull request merged**